build.gradle

```diff
@@ -406,6 +409,8 @@ project(':iceberg-flink-runtime') {
     compile(project(':iceberg-nessie')) {
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
     }
+
+    compile "org.apache.flink:flink-connector-base"
```

Comment:
This jar was not included into the flink-dist binary package, so we have to include it into iceberg-flink-runtime jar explicitly ?

Reply:
that is correct